en/web-interface/system/yeti-ui-system-api_log_configs.rst

```diff
@@ -4,13 +4,18 @@
 Api Log Configs
 ~~~~~~~~~~~~~~~
 
-Api Log Configs are used for setting (where necessary) Debug mode for storing records to the :ref:`API Log <api_logs>`.
-You can click on the  "Yes/No" button near the records for changing their state.
+Api Log Configs are used for setting (where necessary) Debug mode for storing additional info to the :ref:`API Log <api_logs>`.
+You can create specific API Log Configs or delete them from Admin UI.
+
+The Api Log Configs is a part of our system that enables the configuration of :ref:`API logging <api_logs>` behavior.
+It defines how API requests and responses are logged and whether debugging information is captured for specific API controllers.
 
 **Api Log Config**'s properties:
 ````````````````````````````````
     Controller
         Controller that will be used for applying of the Debug mode of storing records to the :ref:`API Log <api_logs>` in case of enabling Debug property bellow.
-    Debug
-        In case of enabling this property (by clicking on it) additional fields (Request Body, Response Body, Request Headers, Response Headers) will be stored to the :ref:`API Log <api_logs>` for the Controller above.
 
+Best Practices:
+```````````````
+Use API log configurations judiciously to capture logs only for controllers that require detailed logging.
```

Comment:
@Ivanov-Anton  maybe better to use rationally instead of judiciously

Reply:
@Fivell, "Yes, it sounds better"